py/tests/integration/test_use_cases.py

```diff
@@ -176,3 +217,22 @@ def node_peer_url(node_name):
     host = common.node_config(node_name)["host"]
     port = common.node_config(node_name)["ports"]["external_api"]
     return "http://" + host + ":" + str(port) + "/"
+
+def make_fast_mining_config(root_dir, file_name):
+    sys_config = os.path.join(root_dir, file_name)
+    f = open(sys_config, "w")
+    # if autostart is not true - there will be no miner
+    conf ='[{aecore, [{autostart, true},' + \
+                    ' {aec_pow_cuckoo, {"lean16", "-t 5", 16}}]}].' 
```

Comment:
Does this mean that https://github.com/aeternity/epoch/blob/ccbc61fa6e938661bc0c7d238624f54d4fe41fb0/py/tests/release.py#L20 can be deleted?